tests/unit/test_quiz.py

```diff
@@ -340,3 +340,12 @@ def test_questions():
     assert len(list(questions.get_all())) == 2
     assert questions[1].index == 1
     assert questions[2].engine.dsn.name == "test_dummy_path"
+
+
+def test_set_dummy_tables_path_in_debug_context():
+    with debugger.activate_debug_context(
+        dummy_tables_path="foo", render_function=lambda v: v
+    ):
+        questions = quiz.Questions()
+        questions.set_dummy_tables_path("bar")
+        assert debugger.DEBUG_QUERY_ENGINE.dsn.name == "bar"
```

Comment:
Worth an assert that `debugger.DEBUG_QUERY_ENGINE` gets reset outside the context manager?